partitura/io/exportmusicxml.py

```diff
@@ -834,6 +834,17 @@ def do_harmony(part, start, end):
             bass_step_e = etree.SubElement(bass_e, "bass-step")
             bass_step_e.text = h.bass
         result.append((h.start.t, None, harmony_e))
+
+    # Does harmony annotation for cadences
+    # TODO: Merge with existing Roman Numeral and ChordSymbol annotations if they exist.
+    harmony = part.iter_all(score.Cadence, start, end)
+    for h in harmony:
+        harmony_e = etree.Element("harmony", print_frame="no")
+        function = etree.SubElement(harmony_e, "function")
+        function.text = "|" + h.text
+        kind_e = etree.SubElement(harmony_e, "kind", text="")
+        kind_e.text = "none"
+        result.append((h.start.t, None, harmony_e))
```

Comment:
Cadences are added to Roman Numeral labels or by themselves as `i | PAC` or `| PAC` as an exported harmony string. I also open to other suggestions.